video/cloud-client/analyze/video_detect_person_beta.py

```diff
@@ -0,0 +1,99 @@
+
+
+def detect_person(local_file_path="path/to/your/video-file.mp4"):
+    """Detects people in a video from a local file."""
+
+    client = videointelligence.VideoIntelligenceServiceClient()
+
+    with io.open(local_file_path, "rb") as f:
+        input_content = f.read()
+
+    # Configure the request
+    config = videointelligence.types.PersonDetectionConfig(
+        include_bounding_boxes=True,
+        include_attributes=True,
+        include_pose_landmarks=True,
+    )
+    context = videointelligence.types.VideoContext(
+        person_detection_config=config
+    )
+
+    # Start the asynchronous request
+    operation = client.annotate_video(
+        input_content=input_content,
+        features=[videointelligence.enums.Feature.PERSON_DETECTION],
+        video_context=context,
+    )
+
+    print("\nProcessing video for person detection annotations.")
+    result = operation.result(timeout=300)
+
+    print("\nFinished processing.\n")
+
+    # Retrieve the first result, because a single video was processed.
+    annotation_result = result.annotation_results[0]
+
+    for annotation in annotation_result.person_detection_annotations:
+        print("Person detected:")
+        for track in annotation.tracks:
+            print(
+                "Segment: {}s to {}s".format(
+                    track.segment.start_time_offset.seconds
+                    + track.segment.start_time_offset.nanos / 1e9,
+                    track.segment.end_time_offset.seconds
+                    + track.segment.end_time_offset.nanos / 1e9,
+                )
+            )
+
+            # Each segment includes timestamped objects that include
+            # characteristic - -e.g.clothes, posture of the person detected.
+            # Grab the first timestamped object
+            timestamped_object = track.timestamped_objects[0]
+            box = timestamped_object.normalized_bounding_box
+            print("\tBounding box:")
+            print("\t\tleft  : {}".format(box.left))
+            print("\t\ttop   : {}".format(box.top))
+            print("\t\tright : {}".format(box.right))
+            print("\t\tbottom: {}".format(box.bottom))
+
+            # Attributes include unique pieces of clothing,
+            # poses, or hair color.
+            print("\tAttributes:")
+            for attribute in timestamped_object.attributes:
+                print(
+                    "\t\t{}:{} {}".format(
+                        attribute.name, attribute.value, attribute.confidence
+                    )
+                )
+
+            # Landmarks in person detection include body parts
+            print("\tLandmarks:")
+            for landmark in timestamped_object.landmarks:
+                print(
+                    "\t\t{}: {} (x={}, y={})".format(
+                        landmark.name,
+                        landmark.confidence,
+                        landmark.point.x,
```

Comment:
are these normalized coordinates too, and (x, y)=(0, 0) at the upper left?  there are different conventions out there so perhaps it is beneficial to include that in the comment here.

Reply:
This I'm not sure of and I'll look into it

Reply:
Yes, going to add this comment to both person samples: https://github.com/googleapis/java-video-intelligence/blob/master/proto-google-cloud-video-intelligence-v1p3beta1/src/main/proto/google/cloud/videointelligence/v1p3beta1/video_intelligence.proto#L445